packages/checks/src/is-prototype-polluted.ts

```diff
@@ -0,0 +1 @@
+export const isPrototypePolluted = (key: string): Boolean => ['__proto__', 'prototype', 'constructor'].includes(key);
```

Comment:
Could you please change this to using a `Set` for slightly better perf (5-10%):

```ts
const ILLEGAL_KEYS = new Set(["__proto__", "prototype", "constructor"]);

export const isPrototypePolluted = (key: string) => ILLEGAL_KEYS.has(key);
```

Reply:
Otherwise, looks all very good! Thank you again!

Reply:
@postspectacular - committed, looking good?

Reply:
Yes, it does & thank you!